claim/README.md

```diff
@@ -41,44 +38,68 @@ There are two important notes regarding connecting nodes:
 
 ## How to connect a node
 
+There will be two main flows from where you might want to connect a node to Netdata Cloud.
```

Comment:
@hugovalente-pm Only admins can connect nodes for both scenarios. This needs to be mentioned

Reply:
Will add that note, thanks.